### DISCUSSION TITLE
How do I insert newlines using `--body`

### DISCUSSION THREAD

**Discussion by RulerOf** [ankitpokhrel/jira-cli#588] [Q&A] [answered]
I'm trying to create an issue like this:
```
jira issue create --summary "Ticket Summary" --body "Line 1\nLine2\nLine3"
```

This does not work as expected, instead creating a ticket whose body is literally `Line 1\nLine2\nLine3`.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @RulerOf, You need to use [ANCI-C Quoting](https://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html). It is also demonstrated in one of the example in help.

```sh
jira issue create --summary "Ticket Summary" --body $'Line 1\nLine2\nLine3'
```

**Reply by RulerOf** [reply]
Ah! Been using a shell for how long and never knew that was a thing. 

Thanks @ankitpokhrel!
